src/lib.js

```diff
@@ -257,7 +261,7 @@ export default class Migrator {
           name: migrationName,
           createdAt: timestamp
         });
-        return createdMigration.toJSON();
+        return createdMigration;
```

Comment:
@npahucki doing `await MigrationModel.create(...)` is essentially the same as `.then` but with syntactic sugar to make it look synchronous (async/await is like generator/yield pair for resolving promises using co-routines). Anyway, if you pre-pend await it will return a model (`m` in your comment). Calling toJSON then will include the virtuals and remove the migration model's `id` fields, since we don't need the consumer to know about that.

The function will return the created migration and its details (matching filename, date, etc...) for consumption.